### PR TITLE
docs: remove email adress

### PR DESCRIPTION
**Pull request by MateoLostanlen**
Let's avoid to be spam defusing our email adress like this.

Wildfire will remain private and reserved to Pyronear members for now. To make public a good quality dataset is one of the mission we want to undertake but the current conditions do not allow it

### CODE REVIEW

File: references/classification/README.md [hunk 1]
```diff
@@ -19,21 +19,19 @@ pip install pyronear
 
 ## Train
 
-### Wildfire
-
-Download Dataset from https://drive.google.com/file/d/1Y5IyBLA5xDMS1rBdVs-hsVNGQF3djaR1/view?usp=sharing
+### OpenFire
 
-This dataset is protected by a password, please contact us at contact@pyronear.org
+You can also use out opensource dataset without password
```
Comment (frgfm) [inline references/classification/README.md]: typo here: "our open-source dataset"

File: references/classification/README.md [hunk 2]
```diff
@@ -19,21 +19,19 @@ pip install pyronear
...
 ```
-python train.py WildFireLght/ --model rexnet1_0x --lr 1e-3 -b 16 --epochs 20 --opt radam --sched onecycle --device 0
+python train.py OpenFire/ --use-openfire --model rexnet1_0x --lr 1e-3 -b 16 --epochs 20 --opt radam --sched onecycle --device 0
 ```
 
-### OpenFire
+### Wildfire
 
-You can also use out opensource dataset without password
+This dataset is private, you should be part of Pyronear to access it
```
Comment (frgfm) [inline references/classification/README.md]: Suggestion:
"This dataset is private, for now only Pyronear members can access it"

**Review by frgfm — CHANGES_REQUESTED** [review]
Thanks for the PR! Small typo and I made another suggestion

**Review by fe51 — DISMISSED** [review]

File: references/classification/README.md [hunk 3]
```diff
@@ -19,21 +19,19 @@ pip install pyronear
 
 ## Train
 
-### Wildfire
-
-Download Dataset from https://drive.google.com/file/d/1Y5IyBLA5xDMS1rBdVs-hsVNGQF3djaR1/view?usp=sharing
+### OpenFire
 
-This dataset is protected by a password, please contact us at contact@pyronear.org
+You can also use out open-source dataset without password
```
Comment (frgfm) [inline references/classification/README.md]: I think you forgot part of my previous comment : 
"out open-source ..." --> "**our** open-source ..."

Comment (frgfm) [reply references/classification/README.md]: And "without **a** password"

File: references/classification/README.md [hunk 4]
```diff
@@ -19,21 +19,19 @@ pip install pyronear
...
 ```
-python train.py WildFireLght/ --model rexnet1_0x --lr 1e-3 -b 16 --epochs 20 --opt radam --sched onecycle --device 0
+python train.py OpenFire/ --use-openfire --model rexnet1_0x --lr 1e-3 -b 16 --epochs 20 --opt radam --sched onecycle --device 0
 ```
 
-### OpenFire
+### Wildfire
 
-You can also use out opensource dataset without password
+This dataset is private for now, for now only Pyronear members can access it
```
Comment (frgfm) [inline references/classification/README.md]: "for now" is there twice
"This dataset is private, for now only Pyronear members can access it"

**Review by frgfm — CHANGES_REQUESTED** [review]
Some typos left!

File: references/classification/README.md (hunk 3 above)
Comment (MateoLostanlen) [reply references/classification/README.md]: :)

**Comment by MateoLostanlen** [comment]
> 
> 
> Some typos left!

Sorry, I did this too fast :)

**Review by frgfm — APPROVED** [review]
Thanks for the edits!

**Pull request merged**